### PR TITLE
adding logout option, login required for personal pages

### PR DESCRIPTION
**Pull request by erezdav22**
Added logout option, and now there only users who are logged in can go to "personal area" and "available jobs" pages.
If someone who is not logged in tries to access these pages- he is redirected to the sign in page. 
only after he puts in correct username and password- he is redirected to the personal area.

fixes issue #66

### CODE REVIEW

File: ultimatejobweb/templates/sign_in.html [hunk 1]
```diff
@@ -19,7 +19,8 @@
  <form method="post">
   {% csrf_token %}
   {{ form.as_p }}
-  <button type="submit">Log In</button>
+  <input type="submit" value="Login">
+<!--   <button type="submit">Log In</button> -->
```
Comment (simzacks) [inline ultimatejobweb/templates/sign_in.html]: remove old code, don't make it a comment

**Review by simzacks — CHANGES_REQUESTED** [review]
minor fix and its ready to go

File: ultimatejobweb/templates/sign_in.html (hunk 1 above)
Comment (erezdav22) [reply ultimatejobweb/templates/sign_in.html]: Done!

Comment (simzacks) [reply ultimatejobweb/templates/sign_in.html]: Its still here

**Review by simzacks — CHANGES_REQUESTED** [review]

**Pull request merged**